src/main/ncdio_pio.F90.in

```diff
@@ -1541,11 +1541,11 @@ contains
        call ncd_inqvid  (ncid, varname, varid, vardesc)
 
 #if ({DIMS}==0)
+       start(1) = 1 ; count(1) = len(data)
```

Comment:
I just want to check here: You moved the setting of start and count outside the `if (present(nt))` conditional, but you didn't change the `pio_put_var` call on line 1556, so these start and count values are still only used in the `if (present(nt))` block. Was that intentional, or should line 1556 also be changed to add these start & count arguments?

Reply:
I missed that one, thanks.  Updated now.

Reply:
Actually, I'm getting build errors with this latest change, with both pio1 and pio2. So I guess your latest commit should be reverted?